### DISCUSSION TITLE
disk space issues

### DISCUSSION THREAD

**Discussion by omercnet** [mend/renovate-ce-ee#313] [Q&A]
every few weeks the container runs out of disk space and git clone fails - stoping the whole operation

what's the recommended way to handle this?

**Comment by rarkins** [comment]
Can you work out which directories are consuming the most space?

**Reply by omercnet** [reply]
I restarted the deployment to clear everything, it'll take a few days to accumulate again

I added a larger volume to /tmp/renovate - is that where everything is stored?

**Reply by rarkins** [reply]
Yes, it's meant to be in `/tmp/renovate`, which you could map to the host if that makes things easier. But I can't be sure that a package manager isn't building up a cache somewhere which we've missed. It's also good to know which part of the tmp dir is accumulating so much, e.g. is it 90% Gradle cache or something like that.

You can likely get an idea for the amount of use well before you run out btw, no need to wait for a crash
